### DISCUSSION TITLE
Support for geotiff tag decoding

### DISCUSSION THREAD

**Discussion by bradh** [ngageoint/tiff-java#35] [Ideas]
Do you see decoding of the geotiff tags as in scope? It looks like it could be added on (like libgeotiff + libtiff), but interested in whether it might be better within tiff-java.

**Comment by bosborn** [comment]
We have not needed it thus far, so have not implemented it.

Would not be opposed to having the functionality either
* inside this library if done in a non intrusive to base functionality manner
* as a separate library with a dependency

[This tiff ios issue](https://github.com/ngageoint/tiff-ios/issues/8#issuecomment-546929766) provides a reference to the geotiff.js behavior that could be referenced.  Our team currently has no near term plans on implementing.  If someone implements and tests it, we can determine whether this or an alternate library is the best place for it.

**Reply by bradh** [reply]
Thanks. The linked issue is pretty handy, because while I have some of the decoding done (with a loose SPI based implementation), I don't have a good feel for a general API.
